bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/LogAction.java

```diff
@@ -7,6 +7,7 @@
  */
 package org.eclipse.smarthome.model.script.actions;
 
+import org.quartz.utils.FindbugsSuppressWarnings;
```

Comment:
Can we add the spotbugs eclipse plugin to the target platform? It exports the annotations package.